src/EventStore.ClusterNode/EventStore.ClusterNode.csproj

```diff
@@ -36,4 +38,23 @@
     <Copy ContinueOnError="ErrorAndStop" SourceFiles="..\libs\x64\win\js1.dll" DestinationFolder="$(OutDir)" Condition=" '$(OS)' == 'Windows_NT' " />
     <Copy ContinueOnError="ErrorAndStop" SourceFiles="..\libs\x64\win\js1.pdb" DestinationFolder="$(OutDir)" Condition=" '$(OS)' == 'Windows_NT' " />
   </Target>
+
+    <!-- qq Required to get the native assembly e_sqlite3.dll, from Microsoft.Data.Sqlite/SQLitePCLRaw, copied to \bin,
```

Comment:
remove qq

Reply:
this may also need changes to packaging scripts since the build directory has changed